### DISCUSSION TITLE
Is it possible to make percentages in a table based on a relationship using a filament?

### DISCUSSION THREAD

**Discussion by ruswan** [filamentphp/filament#6556] [General]
I have a complaint table like this:
![image](https://github.com/filamentphp/filament/assets/6128359/730e5444-c365-4e25-b8b1-be46852e258d)

![image](https://github.com/filamentphp/filament/assets/6128359/d2666e45-c232-4cc3-8db6-e362ebe4660a)

I want to create report  from scope table  and get percentage based on complaint table. The report I want is something like this
![image](https://github.com/filamentphp/filament/assets/6128359/497c42d1-77de-4eab-bd7c-d3dc81acf59b)

This is the code i have made:

````
 public static function table(Table $table): Table
    {
        return $table
            ->columns([
                Tables\Columns\TextColumn::make('name')
                    ->label('SCOPE'),

                Tables\Columns\TextColumn::make('complaints_count')
                    ->counts('complaints')
                    ->label('TOTAL')
                    ->sortable(),

                Tables\Columns\TextColumn::make('percentage')
                    ->label('PERCENTAGE')
                    ->sortable(),
            ]);
    }

````

And this is is the result

![image](https://github.com/filamentphp/filament/assets/6128359/3cb5652c-8842-427b-8ae7-ba57551f3e67)

**Comment by danharrin** [comment]
Something like this to calculate? https://filamentphp.com/docs/2.x/tables/columns/getting-started#calculated-state

**Reply by ruswan** [reply]
Thank's sir, it really works. You saved my life

![image](https://github.com/filamentphp/filament/assets/6128359/7858aa0c-42be-4a21-af66-07b469238f4a)
